tensorflow/core/kernels/conv_ops_fused_int8.cc

```diff
@@ -261,7 +261,7 @@ struct LaunchFusedConv2DOpCpuInt8Helper {
         }
 
         // (2) Side input.
-        if (side_input_scale != 0.0f) {
+        if (side_input_scale != 0.0f && side_input_base != nullptr) {
```

Comment:
Now this will be a silent error and an additional condition check in a tight loop (perf degradation).

Instead, we should validate this as soon as it is initialized (line 208), return an error status (requires refactoring function to actually return Status instead of silently failing).